docs/helpers/Detox.md

```diff
@@ -118,8 +118,8 @@ I.click({ ios: 'Save', android: 'SAVE' }, '#main'); // different texts on iOS an
 
 #### Parameters
 
--   `locator` **([string][6] \| [object][7])** 
--   `context` **([string][6] \| [object][7])**  (optional, default `null`)
+-   `locator` **CodeceptJS.LocatorOrString** 
```

Comment:
This shouldn`t merge to master. 
Because it is not clear what the Object =) 
Please, make rebase by master and generate typing again.